src/OctoshiftCLI.Tests/ado2gh/Commands/MigrateRepoCommandTests.cs

```diff
@@ -107,6 +109,69 @@ public async Task Happy_Path()
             mockLogger.VerifyNoOtherCalls();
         }
 
+        [Fact]
+        public async Task Idempotency_Stop_If_Target_Exists()
+        {
+            // Arrange
+            const string adoOrg = "FooOrg";
+            const string adoTeamProject = "BlahTeamProject";
+            const string adoRepo = "foo-repo";
+            const string githubOrg = "foo-gh-org";
+            const string githubRepo = "gh-repo";
+            var adoRepoUrl = $"https://dev.azure.com/{adoOrg}/{adoTeamProject}/_git/{adoRepo}";
```

Comment:
The below warning (useless assignment) are legit as far as I can see and need to be addressed.